### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by nazywam**

### CODE REVIEW

**Comment by nazywam** [comment]
Do we want to add the deploy stage here as well?

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
```
Comment (psrok1) [inline .github/workflows/python-app.yml:14]:
```suggestion
```

File: .github/workflows/python-app.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    runs-on: ubuntu-latest
+
```
Comment (psrok1) [inline .github/workflows/python-app.yml:16]:
```suggestion
```

File: .github/workflows/python-app.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
```
Comment (psrok1) [inline .github/workflows/python-app.yml:4]:
```suggestion
name: Lint and test on Python 3.6
```

**Pull request merged**